doctr/models/recognition/master/pytorch.py

```diff
@@ -0,0 +1,290 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import math
+import torch
+from torch import nn
+from torch.nn import functional as F
+from typing import Dict, Any, Tuple, Optional, List
+
+from ....datasets import VOCABS
+from ...utils import conv_sequence_pt
+from ...backbones import resnet_stage
+from .transformer_pt import Decoder, positional_encoding, create_look_ahead_mask, create_padding_mask
+from .base import _MASTER
+
+__all__ = ['MASTER']
+
+
+default_cfgs: Dict[str, Dict[str, Any]] = {
+    'master': {
+        'mean': (.5, .5, .5),
+        'std': (1., 1., 1.),
+        'input_shape': (3, 48, 160),
+        'post_processor': 'MASTERPostProcessor',
```

Comment:
You can remove this (since it's instantiated in the model by default, the information is no longer useful)